src/compound/SupplyVaultBase.sol

```diff
@@ -92,17 +92,15 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe, OwnableUpgradeable
     /// PUBLIC ///
 
     function totalAssets() public view override returns (uint256) {
-        IMorpho morphoMem = morpho;
```

Comment:
Why removing it? Is it no longer saving gas?

Reply:
it's an immutable variable so no need to do that